### PR TITLE
Update and correct PT-BR translations

### PR DESCRIPTION
**Pull request by lucasmz-dev**
I've had some issues with Git in the process (broke my brain a little bit, but ok) hopefully that isn't a problem; Git is still something I get confused by :P

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -62,18 +62,18 @@
...
     <string name="lblExtendedDatabaseToggle">Banco de dados extendido</string>
     <string name="confirm_extended_title">Ativar o banco de dados extendido?</string>
-    <string name="confirm_extended_summary">[EXPERIMENTAL]\nIsto ativará a detecção de ~40 milhões de assinaturas adicionais.\nIsto precisará de um download de ~125MB, irá fazer com que a inicialização demore mais de 2 minutos, aumentará o uso de RAM do app, e aumentará a taxa de falsos positivos.\nEste banco de dados atualiza somente trimestralmente.</string>
+    <string name="confirm_extended_summary">[EXPERIMENTAL]\nAtivará a detecção de ~40 milhões de assinaturas adicionais.\Requer um download de ~125MB, irá fazer com que a inicialização demore mais de 2 minutos, aumentará o uso de RAM do app, e aumentará a taxa de falsos positivos.\nEste banco de dados atualiza somente trimestralmente.</string>
     <string name="confirm_update_title">Confirmar download</string>
-    <string name="confirm_update_summary">Você parece estar em uma conexão medida. Você tem certeza que deseja atualizar os bancos de dados?\nIsso pode precisar de %s megabytes de dados.</string>
```
Comment (LucasMZReal) [inline app/src/main/res/values-pt-rBR/strings.xml:75]: Accidentally removed the /n, correcting

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -62,18 +62,18 @@
...
     <string name="delete_failed">Falha ao excluir!</string>
     <string name="ignored">Ignorado!</string>
     <string name="uninstallApp">Desinstalar</string>
-    <string name="lblSelfTest">Escrever arquivos de autoverificação</string>
+    <string name="lblSelfTest">Criar arquivos de autoverificação</string>
     <string name="lblExtendedDatabaseToggle">Banco de dados extendido</string>
     <string name="confirm_extended_title">Ativar o banco de dados extendido?</string>
-    <string name="confirm_extended_summary">[EXPERIMENTAL]\nIsto ativará a detecção de ~40 milhões de assinaturas adicionais.\nIsto precisará de um download de ~125MB, irá fazer com que a inicialização demore mais de 2 minutos, aumentará o uso de RAM do app, e aumentará a taxa de falsos positivos.\nEste banco de dados atualiza somente trimestralmente.</string>
+    <string name="confirm_extended_summary">[EXPERIMENTAL]\nAtivará a detecção de ~40 milhões de assinaturas adicionais.\Requer um download de ~125MB, irá fazer com que a inicialização demore mais de 2 minutos, aumentará o uso de RAM do app, e aumentará a taxa de falsos positivos.\nEste banco de dados atualiza somente trimestralmente.</string>
```
Comment (lucasmz-dev) [inline app/src/main/res/values-pt-rBR/strings.xml]: Removed the \n accidentally, fixing

**Comment by lucasmz-dev** [comment]
Wondering slightly if this means all these commits just show up when pulled... I've seen pull requests where only one is created, even if there were some. If it is better, I'll create a new one.

**Comment by SkewedZeppelin** [comment]
Thank you! I'll merge this with a batch of others soon.

**Comment by lucasmz-dev** [comment]
@SkewedZeppelin if you see new strings being added while Weblate isn't a thing yet, let me know if you want as I can translate them for you. Applies to any of the apps I translated for DCG.

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**